community/samples/serving/helloworld-deno/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM hayd/alpine-deno:0.38.0
```

Comment:
Should this use `1.0.0-rc2`?
We prefer not to have patch-level versions in the container tag so bugfixes come through on docker pull. However, upstream doesn't provide that, so 🤷

Reply:
I believe the base image is fine as-is.